README.md

```diff
@@ -67,6 +67,7 @@ See a more detailed example with walk-through in the [example folder](./example)
 | <a name="output_network_id"></a> [network\_id](#output\_network\_id) | n/a |
 | <a name="output_ssh_private_key"></a> [ssh\_private\_key](#output\_ssh\_private\_key) | Key to SSH into nodes |
 | <a name="output_subnet_id"></a> [subnet\_id](#output\_subnet\_id) | n/a |
+| <a name="output_server_locations"></a> [server\_locations](#output\_server\_locations) | Array of hetzner server locations we deploy to |
```

Comment:
You should not write those by hand! Run `pre-commit run -a`. Or do `pre-commit install` do add it as automatic hook before each commit.